### PR TITLE
Upgrade swagger-ui to version 5.1.3

### PR DESCRIPTION
**Pull request by justin-tay**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: .factorypath [hunk 1]
```diff
@@ -0,0 +1,212 @@
+<factorypath>
```
Comment (moizarafat) [inline .factorypath:1]: was this file intended to be checked in?

Comment (justin-tay) [reply .factorypath]: Yes this was unintended. Have removed it.

**Review by aklish — APPROVED** [review]
